topomodelx/base/aggregation.py

```diff
@@ -41,6 +41,8 @@ def update(self, inputs):
             return torch.sigmoid(inputs)
         if self.update_func == "relu":
             return torch.nn.functional.relu(inputs)
+        if self.update_func == "tanh":
+            return torch.tanh(inputs)
```

Comment:
In the original paper by Roddenberry et al. where SCoNe is proposed, the authors prove that it is necessary for the activitaion function to be odd and non-linear. Based on this result, they choose to use the hyperbolic tangent activation function ($\tanh$).

I have added $\tanh$ as a possible choice for the update function in the `Aggregation` class.